ci-operator/step-registry/gather/audit-logs/gather-audit-logs-commands.sh

```diff
@@ -6,4 +6,7 @@ then
 	exit 0
 fi
 
-oc adm must-gather --dest-dir="$ARTIFACT_DIR" -- /usr/bin/gather_audit_logs
+dest_dir=$(mktemp --dir --tmpdir="$ARTIFACT_DIR")
```

Comment:
This does not need to be random; we get a fresh directory for this step.  How about:

```sh
mkdir "${ARTIFACT_DIR}/audit-logs"
oc adm must-gather --dest-dir="${ARTIFACT_DIR}/audit-logs" -- /usr/bin/gather_audit_logs
```

and skip the `mv` and `rm`?

Reply:
@wking I have updated to match the must-gather task.